packages/ui/src/components/theme/styleOverrides/utils/getContainerHeight.ts

```diff
@@ -0,0 +1,27 @@
+export const TOOLBAR_HEIGHT = 4;
+export const TOOLBAR_HEIGHT_SM = 3.5;
+export const PAGE_HEADER_BREADCRUMBS_HEIGHT = 6.87;
+export const PAGE_HEADER_HEIGHT = 4.87;
```

Comment:
I would use these values as defaults in order to make this util function extensible by the clients. For that I would make `getContainerHeight` receive a second arg, called `options` which would be an object of the shape:

```
{
  toolbarHeight: '4rem',
  toolbarHeightSm: '3.5rem',    // give the unit of measure as well to allow the client of the library to input theirs
  ....
}
```